homeassistant/components/backup/manager.py

```diff
@@ -494,16 +541,18 @@ async def _async_create_backup(
         backup_id: str,
     ) -> BaseBackup:
         """Generate a backup."""
+        manager = self._hass.data[DATA_MANAGER]
         success = False
 
         local_file_paths = [
-            self.local_backup_agents[agent_id].get_backup_path(backup_id)
+            manager.local_backup_agents[agent_id].get_backup_path(backup_id)
             for agent_id in agent_ids
-            if agent_id in self.local_backup_agents
+            if agent_id in manager.local_backup_agents
         ]
```

Comment:
I think this should happen in the manager instead, and we let the manager copy the tar file to the additional local backup agents

Reply:
Moved to the manager